### ISSUE TITLE
Add option to filter build output

### ISSUE THREAD

**Issue by Schweber** [maralorn/nix-output-monitor#139]
I just started using flakes and am annoyed that the option `--no-build-output` isn't accepted when using flakes.

I like the build graph of `nom` very much but i don't want the hundreds of lines that are printed when something like `xwayland` is compiled.

Can you please add an option that emulates `--no-build-outputs` and just shows the build graph of `nom` (what `nixos-rebuild` is doing at the moment without printing it to stdout persistently)?

**Comment by maralorn** [comment]
Yes, that is a totally reasonable request and nom already has that mode. It is just not exposed on the command line right now.
